blockbuilder/genesisState.go

```diff
@@ -0,0 +1,33 @@
+package blockbuilder
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/validator"
+)
+
+type GenesisConfig struct {
+	ChainID                 string           `json:"chain_id" validate:"required"`
```

Comment:
I see there was some discussion about testing unmarshalling/validation. We test these things in the rpc package. Should we test this validation here? Some basic cases would be enough to catch the problem with unmarshaling `map[felt.Felt]genesisContractData`, for example.

Reply:
> We test these things in the rpc package. Should we test this validation here

We do that because we are trying to replicate an existing schema already. So, testing unmarshaling some sample data we got from another source makes sense. But since we are trying to define a new schema here, handwriting some JSON samples and testing unmarshaling against them doesn't really serve any purpose IMO.

Reply:
Feel free to write tests when you deem necessary, more tests don't hurt.

Reply:
Closing this because I don't think we need marshalling or validation tests just yet